tests/System.IO.Pipelines.Tests/TestBufferFactory.cs

```diff
@@ -0,0 +1,93 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using System.Collections.Generic;
+using System.IO.Pipelines.Testing;
+using System.Linq;
+using System.Text;
+
+namespace System.IO.Pipelines.Tests
```

Comment:
This is useful not on only for pipelines, why is it moved back?

Reply:
It's used just by pipelines tests

Reply:
No, see https://github.com/dotnet/corefxlab/blob/master/tests/System.Buffers.Primitives.Tests/ReadableBufferFacts.cs#L21

Reply:
Not sure why it builds but it's used by ROB and RBR tests that are in primitives.

Reply:
There are two copies. I will remove one of them (i.e. the one in pipelines)